index.html

```diff
@@ -0,0 +1,99 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <link rel="stylesheet" href="style.css">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>CV of Evgeni Borovik</title>
+</head>
+
+<body>
+    <header>
+        <h1>Evgeni Borovik</h1>
+        <hr>
+        <div class="photo">
+            <img src=".\assets\img\photo.jpg" alt="Evgeni Borovik photo">
+        </div>
+    </header>
+    <section class="contacts">
+        <h2 class="header-contacts">Contact information</h2>
+        <p class="paragraph"><em>email:</em> <a href="mailto:anyuser@mail.ru">anyuser@mail.ru</a>
+            <br>
+            <em>skype:</em> eugene_borovik
+            <br>
+            <em>mobile / telegram / viber:</em> <a href="tel:+375296829132">+375 29 682-91-32</a>
+        </p>
+
+    </section>
+    <hr>
+    <section>
+        <h2>Summary</h2>
+        <p class="paragraph">Current position: Strategic Planning Director
+            <br>
+            Target position: Project Manager in a software development company
+        </p>
+    </section>
+    <hr>
+    <section>
+        <h2>Skills</h2>
+        <ul>
+            <li>Project management</li>
+            <li>HTML/CSS basics</li>
+            <li>Git basics</li>
+            <li>Networking basics</li>
+            <li>Digital marketing</li>
+            <li>Research data analysis</li>
+            <li>Excel and PowerPoint proficient user</li>
+        </ul>
+    </section>
+    <hr>
+    <section>
+        <h2>Work experience</h2>
+        <p class="paragraph dates"><em>2013 - until present</em></p>
+        <h3>Strategic Planning Director at “InMedias”</h3>
+        <ul>
+            <li>Communication with key clients</li>
+            <li>Project management</li>
+            <li>Market research</li>
+            <li>Media strategy development (online and offline media)</li>
+            <li>Supervision of Planning department</li>
+        </ul>
+        <p class="paragraph dates"><em>2007 - 2013</em></p>
+        <h3>Media Director at “Salmon Media”</h3>
+        <ul>
+            <li>Media and communications strategy development</li>
+            <li>Media market analysis</li>
+            <li>Personnel training</li>
+        </ul>
+
+
+    </section>
+    <section>
+        <hr>
```

Comment:
https://monosnap.com/file/pJztuWXWGIepK5RTWAVop3ElhoAQUY,
Please write code in one way, I mean if you write hr tag as first element in section write it everywhere,
take a look at screenshot, somewhere it first element in section, somewhere it after the section